packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -265,6 +264,8 @@ export const FilterButton = forwardRef(function FilterButton<
               role='listbox'
               aria-label={selectedLabel ?? label ?? props['aria-label']}
               aria-activedescendant={selectedLabel}
+              css={{ maxHeight: popupProps?.css?.maxHeight, overflowY: 'auto' }}
```

Comment:
idk how I feel about this. Maybe `FilterButton` should accept a maxHeight prop instead?

Reply:
ah yeah i think this is why i went with the popupProps thing. yeah i think maxPopoverHeight would be good, or we add popupProps.maxHeight adn just destructure it out?